src/com/goide/GoAnnotator.java

```diff
@@ -133,6 +137,12 @@ private static TextAttributesKey getColor(GoFieldDefinition o) {
               : GoSyntaxHighlightingColors.STRUCT_LOCAL_MEMBER;
   }
 
+  private static TextAttributesKey getColor(GoFieldNameImpl o) {
+    return StringUtil.isCapitalized(o.getName())
```

Comment:
As far as I understand it's `com.intellij.psi.impl.PsiElementBase#getName`, so I believe that condition will return false.